content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -38,18 +47,76 @@ and
-Without using the BOM, you would need to go and update the dependencies in your plugin to match those required by Jenkins, and then keep changing these as your `jenkins.version` changes.
-With the BOM all you have to do is import the link:https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml[jenkins-bom] for the version of Jenkins you are building against, and the versions of these and other common dependencies will be matched to that version of Jenkins.
+See the video below for a step by step guide to updating the parent pom.
 
-To use the Jenkins Core BOM in your plugin, just use plugin-pom (4.0 or later).
+.Update the parent pom
+video::Fev8KfFsPZE[youtube, width=852, height=480,start=807]
 
-== Jenkins Plugin BOM
+== Jenkins plugin BOM
 
-See the link:https://github.com/jenkinsci/bom[jenkinsci/bom] repository
+The plugin bill of materials (BOM) defines the versions of various plugins used by specific Jenkins versions.
+The plugin bill of materials includes many commonly used plugins
+and simplifies dependency management for plugins that depend on other plugins.
+
+A plugin that tests its Pipeline implementation typically depends on several Pipeline plugins.
+The plugin BOM provides Pipeline plugin version numbers so that the developer does not need to maintain the independent version numbers of the Pipeline plugins.
+
+The plugin bill of materials is defined in the `dependencyManagement` section of your _pom.xml_ file.
+To use the Jenkins plugin BOM in your plugin:
+
+. Confirm that you are using the link:/doc/developer/plugin-development/updating-parent/[current parent pom] with the command `mvn versions:update-parent`
+. Locate the most recent plugin bom version nuumber on the link:https://github.com/jenkinsci/bom/releases[plugin bom releases page]
+. Insert the `dependencyManagement` section into the _pom.xml_ file
++
+[source,xml]
+----
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.jenkins.tools.bom</groupId>
+            <artifactId>bom-2.303.x</artifactId>
+            <!-- Latest version goes here -->
+            <version>1117.v62a_f6a_01de98</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+----
+. Remove the `<version>` values from plugin dependencies in _pom.xml_
+. Rebuild your plugin to see if this solves your dependency issues.
+  Many commonly used plugins are included in the plugin pom.
+  However, not **all** plugins are included.
+  You may need to iterate on removing `<version>` values from specific plugins
+
+See the link:https://github.com/jenkinsci/bom#readme[plugin BOM documentation] for further details.
+The video below demonstrates adding the plugin BOM to a Jenkins plugin.
+
+.Simplify plugin dependency management with the plugin BOM
+video::pk1gweLvcEI[youtube, width=852, height=480, start=1771]
```

Comment:
Not able to see this video as well. Am I missing something from my end?

Reply:
> Not able to see this video as well. Am I missing something from my end?

Not sure.  

Can you describe the environment where you're not seeing the video?  

What operating system?  

What browser?  

What's the URL of the page when you're viewing it?  

Does the video thumbnail image appear on the page?